Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -146,302 +172,303 @@ function GetBiographyTextTable( individualID:number )
-        instance.ClassName:SetHide( noneAvailable );
-        instance.TitleLine:SetHide( noneAvailable );
-        instance.IndividualName:SetHide( noneAvailable );
-        instance.EraName:SetHide( noneAvailable );
-        instance.MainInfo:SetHide( noneAvailable );
-        instance.BiographyBackButton:SetHide( noneAvailable );
-        instance.ClaimedLabel:SetHide( not noneAvailable );
-        instance.BiographyArea:SetHide( true );
-        instance.RecruitInfoArea:SetHide( true );
-        instance.FadedBackground:SetHide( true );
-        instance.BiographyOpenButton:SetHide( noneAvailable );
-        instance.EffectStack:CalculateSize();
-        instance.EffectStackScroller:CalculateSize();
-
-        if (CQUI_isPreferredHeightsComputed == false) then
-            --CQUI_preferredRecruitScrollSize = 1000;  -- for quick testing
-            if (CQUI_preferredRecruitScrollSize > (CQUI_screenHeight / 4)) then
-                CQUI_preferredRecruitScrollSize = (CQUI_screenHeight / 4);
-            end
+        CQUI_isPreferredHeightsComputed = true;
+    end
 
-            -- 670 is the default Instance size in the XML... 86 is a number that represents some undocumented thing
-            CQUI_preferredInstanceSize =  670 - 86 + CQUI_preferredRecruitScrollSize;
-            -- CQUI_preferredInstanceSize = 3000 -- for quick testing
-            if (CQUI_preferredInstanceSize > (CQUI_screenHeight - CQUI_instanceMargin)) then
-                -- The instance area cannot be bigger than the screen height minus CQUI_instanceMargin:
-                -- When the Gold/Faith (or Recruit/Pass) buttons are 5px above the PeopleScroller horizontal scroll, 
-                -- the PanelInstance is CQUI_instanceMargin pixels less in height than the PeopleContainer, which is defined as 768 in the XML.
-                -- These adjustments are necessary to properly fit the screen
-                local prevPreferredInstanceSize = CQUI_preferredInstanceSize;
-                CQUI_preferredInstanceSize = CQUI_screenHeight - CQUI_instanceMargin;
-
-                -- Instead of shrinking the recruit scroll size, instead shrink the EffectsStackScroller, as there's typically room to spare in that section
-                -- 240 is the value defined in the XML.  We cannot do a GetSizeY here because subsequent calls to this function
-                -- would update the smaller value, eventually shrinking the control to less than zero.
-                local effectStackScrollerAdjustment = prevPreferredInstanceSize - CQUI_preferredInstanceSize;
-                CQUI_preferredEffectsScrollSize = 240 - effectStackScrollerAdjustment;
-            end
+    -- CQUI : change size
+    -- CQUI DEBUG TEST :
+    -- CQUI_preferredRecruitScrollSize = 300;
+    instance.Content:SetSizeY(CQUI_preferredInstanceSize);
+    instance.EffectStackScroller:SetSizeY(CQUI_preferredEffectsScrollSize);
+    instance.RecruitScroll:SetSizeY(CQUI_preferredRecruitScrollSize);
+end
```

Comment:
** -- stop scrolling -- **
all of the code from the point above to here is what used to appear in the for loop starting at line 466 below.